testsuite/MDAnalysisTests/test_atomgroup.py

```diff
@@ -275,24 +295,17 @@ def test_newAtomGroup(self):
         # check any special method
         assert_almost_equal(newag.total_mass(), 40.044999999999995)
 
-    
-    # INVALID: there is no such property `_atoms` for an AtomGroup
-    # could check that `self.universe.atoms.ix[0]` is same as `self.universe.atoms[0].ix`
-    @skip
+
     def test_getitem_int(self):
-        assert_equal(self.universe.atoms[0], self.universe.atoms._atoms[0])
+        assert_equal(self.universe.atoms[0].ix, self.universe.atoms.ix[0])
 
-    # INVALID: see above for alternative test
-    @skip
     def test_getitem_slice(self):
```

Comment:
I can change that. But surprisingly `assert_equal` actually does the correct thing and calls `assert_array_equal` when you look into the numpy code.